_posts/2020-03-31-hhvm-4.51.markdown

```diff
@@ -0,0 +1,41 @@
+---
+title: "HHVM 4.51"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.51 is released! This release marks the end of support for 4.45;
+HHVM 4.46&ndash;4.50 remain supported, as do the 4.8 and 4.32 LTS releases.
+
+# Highlights
+
+- A new method `getEnumUnderlyingType()` was added to `ReflectionClass`. It
+  returns the underlying type of the enum's values (usually `int` or `string`).
```

Comment:
Can we say something like?
Usually `HH\int`, `HH\string` or the name of the backing enum.

Reply:
Sure, I can mention that it can be another enum.

HH\int etc.: For auto-imported types I think we prefer (and recommend) always referencing them without HH\

Reply:
https://github.com/facebook/hhvm/commit/c55825e324190dd1ebc20d2097a509c40278f64d#diff-91ccc95efd7b1a8483926f99b51b7c3a

Reply:
Oh I see, the method actually returns them with HH\, I guess in that case it makes more sense to change that here too.